lib/analyzer/static-analyzer.ts

```diff
@@ -104,18 +109,28 @@ export async function analyze(
 
   const appScan = !isTrue(options["exclude-app-vulns"]);
   const nodeModulesScan = !isTrue(options["exclude-node-modules"]);
+  // A runtime logic enabler flag. Is off by default.
+  const collectApplicationFiles = isTrue(options["collect-application-files"]);
 
   if (appScan) {
     staticAnalysisActions.push(
       ...[
         getNodeAppFileContentAction,
+        getNodeJsTsAppFileContentAction,
         getPhpAppFileContentAction,
         getPoetryAppFileContentAction,
         getPipAppFileContentAction,
+        getPythonAppFileContentAction,
         getJarFileContentAction,
         getGoModulesContentAction,
       ],
     );
+
+    if (collectApplicationFiles) {
+      staticAnalysisActions.push(
+        ...[getNodeJsTsAppFileContentAction, getPythonAppFileContentAction],
```

Comment:
You can do `staticAanalysisActions.push(getNodeJsTsAppFileContentAction, getPythonAppFileContentAction)`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push